docs/guides/continuous-integration/ci-provider-examples.mdx

```diff
@@ -179,5 +179,5 @@ and
 
 ### webapp.io
 
-- [cypress-example-layerci](https://github.com/bahmutov/cypress-example-layerci)
-- [Using cypress with webapp.io](https://webapp.io/docs/integrations/cypress)
+- [Basic Example (Layerfile)](https://github.com/webappio/react-e2e-example/blob/master/Layerfile)
```

Comment:
The whole point of me asking for this update was so that we don't have to specify --ci-build-id on the cypress command, but if I upgrade to 13.6.5 and remove the flag in Webapp.io, I get the error:
```
{
  "message": "CI Build ID could not be generated and was not specified.",
  "code": "INDETERMINATE_CI_BUILD_ID"
}
```

@jennifer-shehane Could we perhaps setup a cloud account for Webapp.io for testing purposes and then @joshuadsouza could update the example above to use the --record flag and thus verify that this integration works as expected with Cypress Cloud? 

I'm really trying to remove the use of --ci-build-id because it makes the command non-deterministic and messes up the content-aware caching I'm trying to setup with Turborepo.

Reply:
@ianwalter Could you open a new issue in the cypress repo? I think we may need to make a Cloud change to address this, but it won't be tracked well as a comment in this PR.

Reply:
Will do, thanks!

Reply:
- resolved through https://github.com/cypress-io/cypress/issues/29175